dashboards/gateway-api-inference-extension/metadata.yaml

```diff
@@ -3,7 +3,7 @@ sample_dashboards:
     category: GKE Inference Gateway
     id: inference-extension-prometheus
     display_name: GKE Inference Gateway Prometheus Overview
```

Comment:
This display name should match the display name in the JSON config

and update the README as well